examples/high_frequency/emc/busbar.py

```diff
@@ -117,18 +117,56 @@
 print(q3d.net_sources("Bar2"))
 print(q3d.net_sources("Bar3"))
 
+# Create Matrix Reduction Operations
+
+# Series of Bar1 and Bar2
```

Comment:
@tizianrot, I guess "Create Matrix Reduction Operations" is the title of this new section so we should follow the correct level for the headers.

```suggestion
# ## Create Matrix Reduction Operations
#
# Series of Bar1 and Bar2
```